app/repositories/SessionRepository.scala

```diff
@@ -180,6 +184,32 @@ class DefaultSessionRepository @Inject() (config: AppConfig, component: Reactive
           Left(DatabaseError)
       }
 
+  def get(key:String): Future[RequestOutcome[ProcessContext]] = {
+
+    find("_id" -> key).map { list =>
+      list.size match {
+        case 0 =>  Left(NotFoundError)
+        case _ => Right(ProcessContext(list.head.process, list.head.answers, list.head.labels, None))
+      }
+    }.recover {
+      case lastError =>
+      logger.error(s"Error $lastError occurred in method get(key: String) attempting to retrieve session $key")
+      Left(DatabaseError)
+    }
+
+  }
+
+  def removeSession(key: String): Future[RequestOutcome[Unit]] = {
```

Comment:
not needed as session repo documents are automatically deleted by mongo after session-timeout seconds